cellprofiler/modules/measuregranularity.py

```diff
@@ -79,50 +81,57 @@ def add_image(self, can_remove=True):
 
         group.append("image_name", cps.ImageNameSubscriber(
                 "Select an image to measure", cps.NONE, doc="""
-            Select the grayscale images whose granularity you want to measure."""))
+                Select the grayscale images whose granularity you want to measure."""))
 
         group.append("subsample_size", cps.Float(
                 "Subsampling factor for granularity measurements",
                 0.25, minval=np.finfo(float).eps, maxval=1, doc='''
-            If the textures of
-            interest are larger than a few pixels, we recommend you subsample the image with a factor
-            &lt;1 to speed up the processing. Down sampling the image will let you detect larger
-            structures with a smaller sized structure element. A factor &gt;1 might increase the accuracy
-            but also require more processing time. Images are typically of higher resolution than is
-            required for granularity measurements, so the default value is 0.25. For low-resolution images,
-            increase the subsampling fraction; for high-resolution images, decrease the subsampling
-            fraction. Subsampling by 1/4 reduces computation time by (1/4)<sup>3</sup> because the size
-            of the image is (1/4)<sup>2</sup> of original and the range of granular spectrum can
-            be 1/4 of original. Moreover, the results are sometimes actually a little better
-            with subsampling, which is probably because with subsampling the
-            individual granular spectrum components can be used as features, whereas
-            without subsampling a feature should be a sum of several adjacent
-            granular spectrum components. The recommendation on the numerical value
-            cannot be determined in advance; an analysis as in this reference may be
-            required before running the whole set.
-            See this <a href="http://www.ravkin.net/presentations/Statistical%20properties%20of%20algorithms%20for%20analysis%20of%20cell%20images.pdf">
-            pdf</a>, slides 27-31, 49-50.'''))
+                If the textures of interest are larger than a few pixels, we recommend
```

Comment:
All of the documentation setting strings spanning multiple lines should be formatted in the following way:

```python
    def create_settings():
        self.my_setting = cellprofiler.setting.MySetting(
            "My setting text",
            my_setting_value,
            doc="""\
This is some documentation about this setting.

To insert a line break between paragraphs, add a blank line.
There should be a blank after *(Used only if...)*, if that appears
in the setting text.

Be sure to include the \-character at the beginning. It prevents
a newline from being added at the beginning of the setting text.
Also, this text is intentionally pushed all the way over to the left.

You can view the full help in the CellProfiler GUI to ensure the
help text renders correctly and without error.
"""
        )
```